draft-ietf-gnap-core-protocol.md

```diff
@@ -6512,11 +6634,11 @@ The privacy considerations in this section are modeled after the list of privacy
 
 Surveillance is the observation or monitoring of an individual's communications or activities. Surveillance can be conducted by observers or eavesdroppers at any point along the communications path.
 
-GNAP assumes the TLS protection used throughout the spec is intact. Without the protection of TLS, there are many points throughout the use of GNAP that would lead to possible surveillance.
+GNAP assumes the TLS protection used throughout the spec is intact. Without the protection of TLS, there are many points throughout the use of GNAP that would lead to possible surveillance. Even with the proper use of TLS, surveillance could occur by several parties throughout the protocol.
```

Comment:
Ok, but there are potential solutions for this (eg mixnets to prevent traffic analysis by an adversary capable of watching the entire network)

Reply:
I was trying to say here that surveillance is still possible in a few ways even if you do tls completely securely, which is what the following sections talk about. It's meant as an introduction, not trying to say that you can break tls to spy on things. Can you think of a better way to get that across?